docs/release/trg-0/trg-7-6.md

```diff
@@ -0,0 +1,61 @@
+---
+title: TRG 7.06 - Legal information for end user content
```

Comment:
Is "end user content" runtime?

Perhaps "Legal information on live/runtime"?

Chatgpt suggested: "Legal Notice Accessibility for End Users"